src/index.ts

```diff
@@ -36,6 +38,8 @@ const windicssModule: Module<UserOptions> = function (moduleOptions) {
     return
   }
 
+  const isViteMode = nuxtOptions.buildModules.includes('nuxt-vite')
```

Comment:
This may cause issues when building site.

Reply:
I second this. You should avoid this check.